### ISSUE TITLE
Add example for Node

### ISSUE THREAD

**Issue by awentzel** [codeclimate/test-reporter#322]
Would like to see an example for Node on Travis CI.

**Comment by amadeu01** [comment]
@awentzel I created a file as I did on my repository for example. I hope it helps you.

**Comment by george-norris-salesforce** [comment]
Yes this would be useful. The setup instructions for this are not very clear.
